cmd/admin-prometheus-metrics.go

```diff
@@ -79,8 +89,17 @@ func printPrometheusMetrics(ctx *cli.Context) error {
 	if e != nil {
 		return e
 	}
+	var metricsSubSystem string
+	switch args.Get(1) {
+	case "node":
+		metricsSubSystem = "node"
+	case "bucket":
+		metricsSubSystem = "bucket"
+	default:
```

Comment:
If we are going to keep specifying the metric type as argument, let's return an error when the user types a random input

Reply:
ok, so not to default to `cluster` and user to pass explicitly

Reply:
Oh ok. Makes sense. Will update..